Remove SBG from the file service

adamdaventryGov opened the pull request:
Remove SBG from the file service

adamdaventryGov committed:
Remove SBG from the file service

Remove SBG from the file service

jonkiky reviewed (approved):
Looks good to me

Nahomtes commented on config.js:
```diff
@@ -54,19 +54,7 @@ const config = {
   mysql_database: process.env.MYSQL_DATABASE,
   session_timeout: process.env.SESSION_TIMEOUT ? parseInt(process.env.SESSION_TIMEOUT) * 1000 : 1000 * 30 * 60,  // 30 minutes
   cookie_secret: process.env.COOKIE_SECRET,
-  // Neo4j Connection
```
Keep the following Config:
  // Neo4j Connection
  neo4j_uri: process.env.NEO4J_URI,
  neo4j_user: process.env.NEO4J_USER,
  neo4j_password: process.env.NEO4J_PASSWORD,

adamdaventryGov committed:
Updates to file service to remove additional unused code

added neo4j back to the config file,removed cloudfront, removed unused libaries in files.js

adamdaventryGov committed:
update to package to add git
